osbrain/tests/test_agent_async_requests.py

```diff
@@ -159,16 +155,15 @@ def test_wait(nsproxy, server_client_late_reply_delay):
 
     # Response received in time
     client.send('async', 1, wait=2)
-    time.sleep(1.1)
-    assert server.get_attr('received') == [1]
-    assert client.get_attr('received') == ['x1']
+    assert wait_agent_attr(server, data=1)
+    assert wait_agent_attr(client, data='x1')
 
     # Response not received in time
     client.send('async', 2, wait=1)
     assert logger_received(logger,
                            log_name='log_history_warning',
                            message='not receive req',
-                           timeout=1.1)
+                           timeout=3)
```

Comment:
Here we are really testing the client waits for 1 second, not 3, so I do not think we should increase this timeout.